events.lic

```diff
@@ -24,13 +25,20 @@ class Flags
     @@matchers[key] = matchers.map { |item| item.is_a?(Regexp) ? item : /#{item}/i }
   end
 
+  def self.add_with_history(key, *matchers)
+    @@flags[key] = false
+    @@matchers[key] = matchers.map { |item| item.is_a?(Regexp) ? item : /#{item}/i }
+    @@history[key] = []
```

Comment:
I'm not exactly sure on the syntax, but it seems reasonable to reuse `self.add(...)` here:

```suggestion
    add(key, matchers)
    @@history[key] = []
```